### DISCUSSION TITLE
Installation

### DISCUSSION THREAD

**Discussion by meminens** [yokoffing/Betterfox#306] [Q&A] [answered]
https://github.com/yokoffing/Betterfox/releases/tag/128.0

Source code file has a lot more files than user.js. Should I copy paste all these file to Firefox profile directory? Or just the user.js?

**Comment by yokoffing** [comment, marked as answer]
Just the user.js. See https://github.com/yokoffing/Betterfox?tab=readme-ov-file#required-reading
